client/http.go

```diff
@@ -38,6 +38,10 @@ func (c client) post(u *url.URL, body interface{}) (*http.Response, error) {
 	}
 	req.Header.Set("Content-Type", "application/json")
 
+	if c.jwt != "" {
```

Comment:
You can decorate the client's Transport and only do this once during client instance creation.

Reply:
done